UID and GID causing permission errors

Running the compose.yml as is (with some minor adjustmenst), I ran into #19 and #30 

With user set as 1000:1000 (or whatever your id you put), when you run `docker compose run bewcloud bash` and check its id, you'll  get below output.

```sh
groups: cannot find name for group ID 1000
I have no name!@e17fd4d18c45:/app$ id
uid=1000 gid=1000 groups=1000
```

If I ran `docker compose run website bash -c "cd /app && make migrate-db"` or start the container, then I get similar to below log

```sh
error: File exists (os error 17), symlink '../commander@4.1.1/node_modules/commander' -> '/app/node_modules/.deno/node_modules/commander'
make: *** [Makefile:20: migrate-db] Error 1
```

When I removed user line from compose.yml, and I could check base user id with `docker compose run website bash id`

```sh
deno@0ea2df04357b:/app$ id
uid=1993(deno) gid=1993(deno) groups=1993(deno)
```

(which is according to [deno's docker image](https://github.com/denoland/deno_docker/blob/2abfe921484bdc79d11c7187a9d7b59537457c31/ubuntu.dockerfile#L20-L22))

So I changed owner of the data-files directory with `sudo chown -R 1993:1993 data-files`. And everything works fine

Installation commands (according to the docker-compose.yml in repo) I used to make things work:

```sh
mkdir data-files
sudo chown -R 1993:1993 data-files # This solves all permission related problems
docker compose up -d  # Start service
docker compose run --rm website bash -c "cd /app && make migrate-db" # Make migrations
```

If you need to have custom user ids, I recommend using similar lines in your dockerfile (that is create new user `bewcloud` with id from .env and group `deno` assigned to it) so users can set id from their env rather than get `I have no name!`

--- a/README.md
+++ b/README.md
@@ -19,8 +19,8 @@ Or on your own machine:
 Download/copy [`docker-compose.yml`](/docker-compose.yml) and [`.env.sample`](/.env.sample) as `.env`.
 
 ```sh
-$ docker compose up # makes the app available at http://localhost:8000
-$ docker compose run website bash -c "cd /app && make migrate-db" # initializes/updates the database (only needs to be executed the first time and on any updates)
+$ docker compose up -d # makes the app available at http://localhost:8000
+$ docker compose run --rm website bash -c "cd /app && make migrate-db" # initializes/updates the database (only needs to be executed the first time and on any updates)
 ```
 
 Alternatively, check the [Development section below](#development).

--- a/README.md
+++ b/README.md
@@ -30,6 +30,9 @@ Alternatively, check the [Development section below](#development).
 > [!IMPORTANT]
 > Even with signups disabled (`CONFIG_ALLOW_SIGNUPS="false"`), the first signup will work and become an admin.
 
+> [!NOTE]
+> `1993:1993` comes from deno's [docker image](https://github.com/denoland/deno_docker/blob/2abfe921484bdc79d11c7187a9d7b59537457c31/ubuntu.dockerfile#L20-L22) where `1993` is the default user id in it.
+
 ## Development
 
 ### Requirements

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ Or on your own machine:
 Download/copy [`docker-compose.yml`](/docker-compose.yml) and [`.env.sample`](/.env.sample) as `.env`.
 
 ```sh
-$ mkdir data-files
+$ mkdir data-files #  local directory for storing user-uploaded files
 $ chown -R 1993:1993 data-files # solves permission related issues
 $ docker compose up -d # makes the app available at http://localhost:8000
 $ docker compose run --rm website bash -c "cd /app && make migrate-db" # initializes/updates the database (only needs to be executed the first time and on any updates)

--- a/README.md
+++ b/README.md
@@ -19,6 +19,8 @@ Or on your own machine:
 Download/copy [`docker-compose.yml`](/docker-compose.yml) and [`.env.sample`](/.env.sample) as `.env`.
 
 ```sh
+$ mkdir data-files
+$ chown -R 1993:1993 data-files # solves permission related issues
 $ docker compose up -d # makes the app available at http://localhost:8000
 $ docker compose run --rm website bash -c "cd /app && make migrate-db" # initializes/updates the database (only needs to be executed the first time and on any updates)
 ```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -5,7 +5,6 @@ services:
     ports:
       - 127.0.0.1:8000:8000
     mem_limit: '256m'
-    user: '${UID}:${GID}' # if you run into issues with permissions for the data-files volume below, see other options at https://stackoverflow.com/a/56904335
     env_file:
       - path: .env
         required: true

